Makefile

```diff
@@ -53,8 +59,8 @@ EPOCH_TEST_COMMIT ?= e68e0e1110e64f906f9b482e548f17d73e02e6b1
 # When this is running in travis, it will only check the travis commit range
 .gitvalidation:
 	@which git-validation > /dev/null 2>/dev/null || (echo "ERROR: git-validation not found. Consider 'make deps' target" && false)
-ifeq ($(TRAVIS),true)
-	@git-validation -q -run DCO,short-subject,dangling-whitespace
-else
 	@git-validation -q -run DCO,short-subject,dangling-whitespace -range $(EPOCH_TEST_COMMIT)..HEAD
-endif
+# ifeq ($(TRAVIS),true)
+# 	@git-validation -q -run DCO,short-subject,dangling-whitespace
+# else
+# endif
```

Comment:
this is temporary while I sort out the travis issue (which I think you found above)